### PR TITLE
KH-170: Add  Renal Tests in NCD Consultation form 

### PR DESCRIPTION
**Pull request by kazlaw**
Link to  ticket [here](https://issues.openmrs.org/browse/KH-170)

### CODE REVIEW

**Comment by kazlaw** [comment]
[OpenMRS (1).webm](https://user-images.githubusercontent.com/36139599/234776441-f25fe0ad-56d3-42c8-aa44-24b368649238.webm)

File: configs/openmrs_config/concepts/labs.csv [hunk 1]
```diff
@@ -12,4 +12,11 @@ a8af7520-1350-11df-a1f1-0026b9348838,,,HbA1c,អេម៉ូក្លូប៊
...
 0bf33271-d553-42ae-a854-624e5258551d,,,Low-density lipoprotein,កូលេស្តេរ៉ុល LDL,LDL,កូលេស្តេរ៉ុល LDL,0,300,mg/dl,,,Test,Numeric,,,,
 a5d68184-6849-4a1d-9a86-5bcea3297f70,,,High-density lipoprotein,កូលេស្តេរ៉ុល HDL,HDL,កូលេស្តេរ៉ុល HDL,1,200,mg/dl,,,Test,Numeric,,,,
-a86c14a1-9453-46af-9b9a-f950b21b8952,,,Triglycerides,ទ្រីក្លីសេរីដ,Triglycerides,ទ្រីក្លីសេរីដ,,,,,,Test,Numeric,,,,
\ No newline at end of file
+a86c14a1-9453-46af-9b9a-f950b21b8952,,,Triglycerides,ទ្រីក្លីសេរីដ,Triglycerides,ទ្រីក្លីសេរីដ,,,,,,Test,Numeric,,,,
+1011AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,,,Creatinine,ក្រេអាទីនីន,Creatinine,ក្រេអាទីនីន,0,,IU/L,,,Test,Numeric,,,,
+681d9109-4414-440f-a733-c4cce954cd41,,,Creatinine Clearance,ការធ្វើតេស្តដើម្បីដឹងពីការច្រោះជម្រះជាតិក្រេអាទីនីនពីក្នុងទឹកនោម,Creatinine Clearance,ការធ្វើតេស្តដើម្បីដឹងពីការច្រោះជម្រះជាតិក្រេអាទីនីនពីក្នុងទឹកនោម,0,,,,,Test,Numeric,,,,
+808d6fc6-f762-4f37-b8ed-ece810f52cef,,,Proteinuria,ជាតិសាច់ក្នុងទឹកនោម (ជាតិប្រូតេអ៊ីនក្នុងទឹកនោម),Proteinuria,ជាតិសាច់ក្នុងទឹកនោម (ជាតិប្រូតេអ៊ីនក្នុងទឹកនោម),0,,,,,Test,Numeric,,,,
```
Comment (rbuisson) [inline configs/openmrs_config/concepts/labs.csv:18]: What is in parentheses in Khmer here?

Comment (rbuisson) [reply configs/openmrs_config/concepts/labs.csv:18]: If that's how we received the translation, then let's proceed.

Comment (kazlaw) [reply configs/openmrs_config/concepts/labs.csv:18]: @rbuisson , it was provided that way [here](https://docs.google.com/spreadsheets/d/13PHkzorfTmYRjuIJv3fCml3zBAuDba6Mgdpu4H7cAg8/edit#gid=0&range=A85:B85)

**Pull request merged**